src/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -1373,7 +1390,95 @@ msgstr ""
 msgid "Anti Ventrue Grinder"
 msgstr ""
 
-#: src/templates/best-cards/generic/master.html:86
+#: src/templates/best-cards/generic/animalism.html:7
+#, python-format
+msgid ""
+"<p> <i>i</i> Provide a strong long-range combat module and good intercept capabilities. Combined with <i>a</i>, it "
+"offers the best intercept in the game and access to the combo card %(read_winds)s. </p>"
+msgstr ""
+"<p> <i>i</i> fournit un module de combat conséquent basé sur la portée. Combiné avec <i>a</i>, cela offre la "
+"meilleure interception du jeu et donne accès à la carte à combo %(read_winds)s. </p>"
+
+#: src/templates/best-cards/generic/animalism.html:19
+msgid ""
+"An excellent combat card. It is not a strike so it cannot be dodged, it is harder to prevent and additional damage "
+"are still inflicted by the strike itself. Moreover, it is ranged damage."
+msgstr ""
+"Une excellente carte de combat. Cela n'est pas une frappe et ne peut donc pas être esquivé, c'est compliqué à "
+"prévenir et les dégâts additionnels sont ensuite infligés par la frappe en elle-même. De plus, cela inflige des "
+"dégâts quelle que soit portée."
+
+#: src/templates/best-cards/generic/animalism.html:27
+msgid ""
+"A versatile defense tool providing either intercept or an unlock effect. Not as surprising as a wake, but being "
+"unlocked can be better."
+msgstr ""
+"Une carte de défense polyvalente fournissant soit de l'interception soit un effet de redressement. Pas aussi "
+"surprenant qu'un réveil, mais être redressé peut être meilleur selon les situations."
+
+#: src/templates/best-cards/generic/animalism.html:34
+#, python-format
+msgid ""
+"An excellent counter against weapons, it is often included in <i>i</i> combat modules. Because <i>i</i> often uses "
+"%(carrion_crows)s and %(aid_from_bats)s for a ranged combat module, a ranged weapon on the opponent side is "
+"especially cumbersome."
+msgstr ""
+"Un excellent contre aux équipements, il est souvent utilisé dans les modules de combat basés sur <i>i</i>. Comme "
+"<i>i</i> utilise souvent %(carrion_crows)s et %(aid_from_bats)s dans leur module de combat utilisant la portée, un "
+"équipement en fournissant chez l'adversaire peut souvent être problématique."
+
+#: src/templates/best-cards/generic/animalism.html:46
+msgid ""
+"The choice between bleed and rush gives it so much versatility, it can be played in most table situations at an "
+"advantage. The fact that the opponent has to choose his strike first is a nice bonus."
+msgstr ""
+"Le choix offert entre la morsure et une action de rush lui donne beaucoup de versatilité, elle peut être jouée dans "
```

Comment:
```suggestion
"Le choix offert entre la morsure et une action de rush lui donne beaucoup de versatilité, elle peut être jouée avantageusement dans la plupart des situations. "
```

Reply:
`la plupart des situations.` is placed on the nextline. py-babel seems to have an autocrop functionnality and sentences are automatically split using `make po`

Reply:
Oui of course - je me suis emballé 😋